### PR TITLE
[OSDOCS-9635]: Update roles for the osd-ccs-admin service account

### PR DESCRIPTION
**Pull request by mletalie**
Version(s):

4.15+
Issue:

https://issues.redhat.com/browse/OSDOCS-9635
Link to docs preview:

https://72303--ocpdocs-pr.netlify.app/openshift-dedicated/latest/osd_install_access_delete_cluster/creating-a-gcp-cluster#osd-create-gcp-cluster-ccs_osd-creating-a-cluster-on-gcp
image

https://72303--ocpdocs-pr.netlify.app/openshift-dedicated/latest/osd_planning/gcp-ccs#ccs-gcp-customer-procedure_gcp-ccs
image

QE review:
- [x] QE has approved this change.

Additional information:

PM, QE, SRE has signed off. Ready for publication.

### CODE REVIEW

**Comment by mletalie** [comment]
Hello. I have been asked to have several reviewers for this PR. these individuals include @lento-sun, @ckandag, @renan-campos, @JakobGray, and @NautiluX. 
Thank you!

**Comment by svmrh** [comment]
Looks good. Thanks @mletalie

**Comment by mletalie** [comment]
SRE has signed off with LGTM per https://redhat-internal.slack.com/archives/CB53T9ZHQ/p1709194777951929?thread_ts=1709013251.809759&cid=CB53T9ZHQ

**Comment by mletalie** [comment]
> /lgtm thx @mletalie

Thanks!

**Review by kcarmichael08 — COMMENTED** [review]
LGTM!

**Review by mletalie — COMMENTED** [review]
/remove-label merge-review-needed

File: modules/osd-create-cluster-ccs.adoc [hunk 1]
```diff
@@ -70,6 +70,8 @@ The project name must be 10 characters or less.
   ** Service Management Administrator
   ** Service Usage Admin
   ** Storage Admin
+  ** Compute Load Balancer Admin
+  ** Role Viewer
```
Comment (NautiluX) [inline modules/osd-create-cluster-ccs.adoc:74]:
```suggestion
  ** Role Viewer
  ** Role Administrator
```

File: modules/ccs-gcp-customer-procedure.adoc [hunk 2]
```diff
@@ -114,6 +114,24 @@ The following roles must be link:https://cloud.google.com/iam/docs/granting-role
...
+
+|Service Account Key Admin
+|`roles/iam.serviceAccountKeyAdmin`
+
+|Service Account Admin
+|`roles/iam.serviceAccountAdmin`
+
+|Service Account User
+|`roles/iam.serviceAccountUser`
+
```
Comment (NautiluX) [inline modules/ccs-gcp-customer-procedure.adoc:134]:
```suggestion
|Role Administrator
|`roles/iam.roleAdmin`
```

File: modules/ccs-gcp-customer-procedure.adoc [hunk 3]
```diff
@@ -114,6 +114,24 @@ The following roles must be link:https://cloud.google.com/iam/docs/granting-role
 |Storage Admin
 |`roles/storage.admin`
 
+|Compute Load Balancer Admin
+|`roles/compute.loadBalancerAdmin`
+
+|Role Viewer
+|`roles/viewer`
+
+|Security Admin
```
Comment (NautiluX) [inline modules/ccs-gcp-customer-procedure.adoc:123]:
```suggestion
|Security Administrator
```

Comment (NautiluX) [inline modules/ccs-gcp-customer-procedure.adoc:123]:
```suggestion
|IAM Security Admin
```
for consistency with OCP docs: https://docs.openshift.com/container-platform/4.15/installing/installing_gcp/installing-gcp-account.html

**Comment by svmrh** [comment]
The updated list (with Role Administrator included per SRE feedback) looks good.

**Pull request merged**